invenio_oauthclient/contrib/cern.py

```diff
@@ -233,10 +233,12 @@ def account_setup(remote, token, resp):
         external_id = res['PersonID'][0]
 
         # Create user <-> external id link.
-        oauth_link_external_id(user, dict(id=external_id, method="cern"))
+        oauth_link_external_id(user, dict(id=external_id, method='cern'))
 
     groups = fetch_groups(res['Group'])
-    session['identity.cern_provides'] = [RoleNeed(group) for group in groups]
+    provides = [UserNeed(user.email)] + \
+               [RoleNeed(group + '@cern.ch') for group in groups]
```

Comment:
can you use `'{0}@cern.ch'.format(group)` instead? you know ... safety first ;-)